### PR TITLE
Blog App Views

### PR DESCRIPTION
**Pull request by preciousakams**
###  Commit details

- created user views
- created post views
- Implemented post count per user
- implemented likes count per user
- implemented comments count per user
- routed all views

### CODE REVIEW

File: app/views/posts/index.html.erb [hunk 1]
```diff
@@ -1 +1,6 @@
-<h1> Displays all posts from a user </h1>
\ No newline at end of file
+<div id="posts">
+<%= render @user %>
+<% @posts.each do |post| %>
+<%= render post %>
```
Comment (Deepakdanger) [inline app/views/posts/index.html.erb:4]: - [ ] In the post index, User should see the  5 most recent comments. Use `five_most_recent_comments` method to show recent comments.

File: app/views/posts/show.html.erb [hunk 2]
```diff
@@ -1 +1,10 @@
...
\ No newline at end of file
+<div class="one-post">
+<h1>Post #<%= @post.id %> By <%= @post.user.name %>  </h1>
+<span class= "counter-span"> 
+<strong>Comments: <%= @post.comments.count %> </strong>
+<strong>Likes: <%= @post.likes.count %> </strong>
+ </span>
+ <div class="post-text">
+    <p><%= @post.text %> </p>
+</div>
```
Comment (Deepakdanger) [inline app/views/posts/show.html.erb:9]: - [ ] In the post show page, User should all the comments related to that post. Please add it here.

**Review by Deepakdanger — CHANGES_REQUESTED** [review]
Hi @kamzzy, 

Good job so far! 👏
There are some issues that you still need to work on to go to the next project but you are almost there!

## To Highlight :100: 
- No linter error :heavy_check_mark: 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Whoistolu — APPROVED** [review]
### Hi @kamzzy 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 

# STATUS: APPROVED 🟢
**Congratulations! 🎉** 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @whoistolu in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**